apps/client/src/components/ContactSentence.tsx

```diff
@@ -1,15 +1,33 @@
-import React from "react";
+import { Paragraph } from "@datapunt/asc-ui";
+import React, { ReactElement } from "react";
 
 import PhoneNumber from "./PhoneNumber";
 
 type Props = {
-  link?: boolean;
   eventName?: string;
+  inline?: boolean;
+  link?: boolean;
+  openingSentence?: string;
 };
 
-export default ({ link = true, eventName }: Props) => (
-  <>
-    Bel dan de gemeente op <PhoneNumber {...{ link, eventName }} />, maandag tot
-    en met vrijdag van 08.00 uur tot 18.00 uur.
-  </>
+type WithParagraphProps = {
+  children: ReactElement<any>;
+  inline: boolean;
+};
+
+const WithParagraph = ({ inline, children }: WithParagraphProps) =>
```

Comment:
Update: since this part is only used with <Alert /> on Conclusion.js. Can you not remove `WithParagraph` to make this component less complex and add update the Alert with:
```
 <Alert
heading="Let op"
content={
  <>    
    <ContactSentence contactSentence="De vergunningcheck is nog in ontwikkeling. Hierdoor kunnen wij nog
    geen zekerheid bieden dat de uitkomst correct is. Ook is de
    informatie nog niet voor iedereen goed te lezen of te beluisteren.
    Wilt u iets zeker weten of wilt u meer informatie? Bel dan" />
  </>
}
/>
```

Reply:
Let's wait with this changed till we implement i18n

Reply:
Fixed!